views/game.hbs

```diff
@@ -2,30 +2,49 @@
 <html>
 
 <head>
-  <meta charset="utf-8" />
+  <meta charset="utf-8" {{#falseOrFalse readyToPlay currentPlayerTurnToPlay}}http-equiv="refresh" content="5"
+    {{/falseOrFalse}} />
   <title>Hex</title>
   <link rel="stylesheet" type="text/css" href="/css/style.css" />
   <link rel="stylesheet" type="text/css" href="/css/game.css" />
 </head>
 
 <body>
-  {{#areEqual state.winner null}}
+  {{#areEqual game.state.winner null}}
   <div class="gameboard">
     <div class="box-current-player">
-      <p>Current player :</p>
-      <p class="current-player" data-player="{{state.turn}}">{{state.turn}}</p>
+      {{#areEqual readyToPlay true}}
+      <p>
+        Current player:
+        <span class="current-player" data-player="{{game.state.turn}}">{{game.state.turn}}</span>
+      </p>
+      {{#areEqual currentPlayerTurnToPlay false}}
+      <p>Waiting for other player to make their move...</p>
+      {{/areEqual}}
+      {{else}}
+      <p>Awaiting player 2...</p>
+      <p>
+        Send <a href="/game/{{game.id}}/join">this link</a> for second player to join
+      </p>
+      {{/areEqual}}
+      <p>
+        Player 1 (white): {{game.player1.username}}
+      </p>
+      <p>
+        Player 2 (black): {{game.player2.username}}
+      </p>
     </div>
-    <div class=" hex-board-frame" style="--rows: {{state.board.length}}">
+    <div class=" hex-board-frame" style="--rows: {{game.state.board.length}}">
       <div class="left-edge"></div>
       <div class="right-edge"></div>
       <div class="top-edge"></div>
       <div class="bottom-edge"></div>
       <div class="hex-board-container">
         <i></i>
-        {{#each state.board}}
+        {{#each game.state.board}}
         {{#each this}}
-        {{#areEqual this.value 'empty'}}
-        <form method="post" action="/game/{{../../id}}">
+        {{#cellCanBePlayed this.value ../../readyToPlay ../../currentPlayerTurnToPlay }}
```

Comment:
these paths (`../../readyToPlay`) in a template are really unnatural. I don't understand how it works. Can't you use the `game` variable?